### PR TITLE
update dmm 2.0.4 & rename to dmmbookviwer

### PR DESCRIPTION
**Pull request by yutarody**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/dmmbookviewer.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+  url "http://dl.aka.dmm.co.jp/dmmviewer/mac/DMMViewerSetup_Mac_#{version}.pkg"
+  name 'DMMbookviewer'
+  homepage 'http://book.dmm.com/info_bookviewer.html'
+  license :gratis
+
+  pkg "DMMViewerSetup_Mac_#{version}.pkg"
+
+  uninstall pkgutil: 'com.dmm.DMMbookviewer',
+            rmdir: '/Applications/DMMbookviewer.app'
```
Comment (vitorgalvao) [inline Casks/dmmbookviewer.rb]: `rmdir` is for empty directories`. Use `delete`.

File: Casks/dmmbookviewer.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+cask 'dmmbookviewer' do
+  version '2.0.4'
+  sha256 'ac794dc71761a8a891dc237c11665866cfe76f7bcce68c20f7afb24a1a4a39b9'
+
+  url "http://dl.aka.dmm.co.jp/dmmviewer/mac/DMMViewerSetup_Mac_#{version}.pkg"
```
Comment (vitorgalvao) [inline Casks/dmmbookviewer.rb]: Needs a `# dmm.co.jp was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

File: Casks/dmmbookviewer.rb (hunk 1 above)
Comment (yutarody) [inline Casks/dmmbookviewer.rb]: Thank you for comment.
This is after the uninstall pkgutil, is the process for which will remain for some reason empty directory(/Applications/DMMbookviewer.app).
I would be better to use the delete command ?

Comment (vitorgalvao) [inline Casks/dmmbookviewer.rb]: Yes. It’s clearer, and there’s no reason not to.

Comment (yutarody) [inline Casks/dmmbookviewer.rb]: Thank you for your advice.
I changed to use delete command.

File: Casks/dmmbookviewer.rb (hunk 2 above)
Comment (yutarody) [inline Casks/dmmbookviewer.rb]: I Insert a that comment

**Pull request merged**